src/account/BiconomySmartAccountV2.ts

```diff
@@ -1367,6 +1370,56 @@ export class BiconomySmartAccountV2 extends BaseSmartContractAccount {
     return nonce
   }
 
+  /**
+   * Transfers ownership of the smart account to a new owner.
+   * @param newOwner The address of the new owner.
+   * @param buildUseropDto {@link BuildUserOpOptions}. Optional parameter
+   * @returns A Promise that resolves to a TransferOwnershipResponse or rejects with an Error.
+   * @example
+   * ```typescript
+   * const walletClient = createWalletClient({
+   *   account,
+   *   chain: baseSepolia,
+   *   transport: http()
+   * });
+   * const smartAccount = await createSmartAccountClient({
+   *   signer: walletClient,
+   *   paymasterUrl: "https://paymaster.biconomy.io/api/v1/...",
+   *   bundlerUrl: `https://bundler.biconomy.io/api/v2/84532/nJPK7B3ru.dd7f7861-190d-41bd-af80-6877f74b8f44`,
+   *   chainId: 84532
+   * });
+   * const response = await smartAccount.transferOwnership(newOwner, {paymasterServiceData: {mode: PaymasterMode.SPONSORED}});
+   * ```
+   */
+  async transferOwnership(
+    newOwner: Address,
+    buildUseropDto?: BuildUserOpOptions
+  ): Promise<TransferOwnershipResponse> {
```

Comment:
This function should return a different payload, to remain consistent with other similar helpers. Can we return this please?

`
type TransferOwnershipResponse = UserOpResponse & { cumulativeGasUsed: number; gasUsed: number; }
`

return it like so:

```
const userOpResponse = await this.sendTransaction(transaction, buildUseropDto);

return {
  cumulativeGasUsed: 0,
  gasUsed: 0,
  ...userOpResponse,
}

```